### DISCUSSION TITLE
how to get Heart On Your Sleeve badge

### DISCUSSION THREAD

**Discussion by ATBKO** [community/community#68923] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

how to get Heart On Your Sleeve badge in github?

**Comment by GalKoaz** [comment]
I think you need to type ❤️ or they remove it for now.

**Comment by eminemahjoub** [comment]
you can earn badges and achievements on GitHub by actively participating in open-source projects, contributing to repositories, creating your projects, and engaging with the GitHub community. These badges are often a recognition of your contributions and activities on the platform.

To stay informed about any new badges or achievements that GitHub may introduce, it's a good idea to regularly check their official documentation and blog, as well as stay active in the GitHub community.

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
